src/resources/nfts/simplehash/index.ts

```diff
@@ -113,3 +114,48 @@ export async function fetchSimpleHashNFTListing(
   );
   return cheapestListing;
 }
+
+/**
+ * Given an NFT, refresh its contract metadata on SimpleHash. If we can't,
+ * refresh metadata for this NFT only.
+ * @param nft
+ */
+export async function refreshNFTContractMetadata(nft: UniqueAsset) {
+  const chain = nft.isPoap
+    ? SimpleHashChain.Gnosis
+    : getSimpleHashChainFromNetwork(nft.network);
+
+  if (!chain) {
+    throw new Error(
+      `refreshNFTContractMetadata: no SimpleHash chain for network: ${nft.network}`
+    );
```

Comment:
Is this caught somewhere? If not, just error log it and return early from the function.